camel/loaders/firecrawl_reader.py

```diff
@@ -167,6 +174,9 @@ def map_site(
             RuntimeError: If the mapping process fails.
         """
         try:
+            if params is None:
+                params = {}
+            params['integration'] = 'camelai'
```

Comment:
Fourth time seeing this exact same code. Definitely needs to be extracted to a helper function.